crates/bevy_app/src/schedule_runner.rs

```diff
@@ -24,18 +29,23 @@ impl Default for RunMode {
     }
 }
 
+/// Configuration information for [`ScheduleRunnerPlugin`].
 #[derive(Copy, Clone, Default)]
 pub struct ScheduleRunnerSettings {
+    /// Determines whether the Schedule is run once or repeatedly.
     pub run_mode: RunMode,
 }
 
 impl ScheduleRunnerSettings {
+    /// Produces settings that tell the ScheduleRunner to run only once.
```

Comment:
I think we should link to the enum variants here and for `run_loop`, which allows us to reduce the repetition.